aces/Loris
# Fix data in fields where HTML characters were double escaped

## Brief summary of changes
This fixes the issue where HTML special characters were escaped twice before being saved causing every save to re-escape the `&` of the previously escaped values.

The problematic characters are `&><"`

Example, if an instrument field contains a `>`:
after first save value becomes `&amp;gt;`
after second save `&amp;amp;amp;gt;`

Causing some values to look like this on projects running cronjobs nightly
`&amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;gt;`

#### Testing instructions (if applicable)

1. before checking out this branch, go to an instrument and save any of the characters listed above.
2. reload the page
3. [optional] save multiple times (make sure to reload the page before re-saving because values do not get refetched from the database automatically in instruments)
4. checkout this branch, and test out the script.
5. end result should show normal characters on the front end. Database values should still be escaped but without duplication

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
